Utils/Accessibility.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+public protocol AccessibilityActionsSource: class {
+    func accessibilityCustomActionsForView(view: UIView) -> [UIAccessibilityCustomAction]?
+}
+
+public class Action {
```

Comment:
I think this class needs a more descriptive name. How about `AccessibilityAction` instead?